packages/@aws-cdk/aws-docdb/lib/cluster.ts

```diff
@@ -164,6 +166,23 @@ export interface DatabaseClusterProps {
    * @default false
    */
   readonly exportAuditLogsToCloudWatch?: boolean;
+
+  /**
+   * The number of days log events are kept in CloudWatch Logs. When updating
+   * this property, unsetting it doesn't remove the log retention policy. To
+   * remove the retention policy, set the value to `Infinity`.
+   *
+   * @default - logs never expire
+   */
+  readonly cloudwatchLogsRetention?: logs.RetentionDays;
```

Comment:
Let's be consistent with the other property names:

```suggestion
  readonly cloudWatchLogsRetention?: logs.RetentionDays;
```